plugins/chat/app/jobs/scheduled/chat/auto_join_users.rb

```diff
@@ -5,14 +5,63 @@ module Chat
     class AutoJoinUsers < ::Jobs::Scheduled
       every 1.hour
 
+      MAX_USERS = 10_000
```

Comment:
I see that there is a `max_chat_auto_joined_users` site setting. Should we depend on that instead or drop that site setting?

Reply:
yeah I will port to use the setting, it is 10k anyway and hidden